docs/samples/calling/app.js

```diff
@@ -666,22 +667,23 @@ async function getMediaStreams() {
   makeCallBtn.disabled = false;
 }
 
-async function addNoiseReductionEffect() {
-  effect = await localAudioStream.getEffect('background-noise-removal');
+async function toggleNoiseReductionEffect() {
+  effect = await localAudioStream.getEffectByKind('noise-reduction-effect');
 
   if (!effect) {
     effect = await Calling.createNoiseReductionEffect(tokenElm.value);
 
-    await localAudioStream.addEffect('background-noise-removal', effect);
-  }
-
-  await effect.enable();
-}
-
-async function removeNoiseReductionEffect() {
-  effect = await localAudioStream.getEffect('background-noise-removal');
-  if (effect) {
-    await effect.disable();
+    await localAudioStream.addEffect(effect);
+    await effect.enable();
+    bnrButton.innerHTML = 'Disable BNR()';
```

Comment:
I don't think we need () in the text. Applies to other places as well

Reply:
Removed